cpp/include/ray/api/wait_result.h

```diff
@@ -0,0 +1,19 @@
+
+#pragma once
+
+#include <ray/core.h>
+#include <vector>
+
+namespace ray {
+namespace api {
+
+class WaitResult {
+ public:
+  std::vector<ObjectID> readys;
+  std::vector<ObjectID> remains;
```

Comment:
```suggestion
  std::vector<ObjectID> remainings;
```

Reply:
Maybe `ready_object_ids` and `unready_object_ids`? I'm not sure if the remaining objects are always unready.

Reply:
Maybe follow the naming in java, ready and unready?